### ISSUE TITLE
Using the checkbox field and Required argument does not work as expected + Paid Support

### ISSUE THREAD

**Issue by zagarskas** [reduxframework/redux-framework#2857]
Support Hash:
http://support.redux.io/?id=YP24Nb07d%2F2vE9IQTlRq405GTGv1AUo6ToNl8O%2Buo%2FRCMqzgn6Ng5fmFyWB8xT5SKvDzNOEYZ2K15ShyD9PEI9fEeXvMk6pl3psFy3NkyfVjC8UPzP%2FTsjJXMRLcM0Qq

Version:  public static $_version = '3.5.9';
PHP 5.3.29 | MySQL 10.0.23 | WordPress 4.4.2

Premium Support purchase (unfulfilled)
Payment:	3111, PayPal, 
Purchase Key e3ae8933363497d531adc3ae0b93bc9d

Problem: Using the checkbox field and Required argument does not work as expected.
Desired result: check box selection with the required argument shows a text field when a checkbox is TRUE and hides a field when a checkbox is FALSE.

Detailed description of Issue with Code and Video:
**VIDEO: **http://www.screencast.com/t/z8c2GGXdC6

I have already reviewed the following, to no avail:
https://github.com/reduxframework/redux-framework/issues/2687
https://github.com/reduxframework/redux-framework/issues/2594
https://github.com/reduxframework/redux-framework/issues/2482
https://github.com/reduxframework/redux-framework/issues/2429

Code:
```

array(
                'id'       => 'product_meta_controls',
                'type'     => 'checkbox',
                'title'    => __( 'Product Metta Control', 'zaga_theme' ),
                'options'  => array(
		'product_age' => 'Show Age levels',
		'product_cat' => 'Show Product Category',
		'product_sub' => 'Show Subjects',
		'product_buy' => 'Show External Link', // should turn ON  product_external_text
		'product_mor' => 'Show Internal Link', 
					),
            ),
		 array('required' => array('product_meta_controls', 'contains', array('product_buy', '==', 1)),
		 'title'     => __( 'Product: External Link button text', 'zaga_theme' ),
                    'id'=>'product_external_text',
                    'type'        => 'text',
                ),

```

Variations tried
```
'required' => array('product_meta_controls', 'contains', array('product_buy', '!=', 0)),
'required' => array('product_meta_controls', 'contains', array('product_buy' => 1)),
// and several items on the issue links above


```

**Comment by kprovance** [comment]
Should be resolved in 3.5.9.4.  Now, I spent WAY longer than an hour on this.  Closer to six.  I think we'll say 'obligation fulfilled'.  Thanks!

**Comment by zagarskas** [comment]
Outstanding, excellent work, Its working for me in 3.5.9.4 
thank you good sir!!!
I would be happy to send you a tip in Paypal, what email should I send $ to?
